src/Tasks/Microsoft.NET.Build.Tasks/CreateAppHost.cs

```diff
@@ -42,7 +44,9 @@ protected override void ExecuteCore()
                     options: new AppHostOptions()
                     {
                         WindowsGraphicalUserInterface = WindowsGraphicalUserInterface
-                    });
+                    },
+                    intermediateAssembly: IntermediateAssembly,
```

Comment:
With this change, I'm wondering if we need to keep AppHostOptions. Seems that would have been more interesting if we were going to do the resources via that. Maybe we should just convert it to a bool?

Reply:
I took your suggestion and removed AppHostOptions.